interfaces/admin_cli.py

```diff
@@ -0,0 +1,453 @@
+        self.return_previous_page()
+
+    def do_manage_plan(self, arg):
+        """
+        Enter PlanMenu for further actions
+        """
+        PlanMenu(self.user).cmdloop()
+
+    def do_manage_volunteer_account(self, arg):
+        """
+        Enter ManageVolunteerMenu for further actions
+        """
+        ManageVolunteerMenu(self.user).cmdloop()
+
+    def do_manage_refugee_profile(self, arg):
+        """
+        Enter ManageCampMenu for further actions
+        """
+        ManageRefugeeMenu(self.user).cmdloop()
+
+    def do_exit(self, arg):
+        super().do_exit(self)
+
+
+class PlanMenu(AdminShell):
+    """
+    PlanMenu is the command line interface for emergency plan management.
+    """
+    plan_options = {"logout": 0, "create_plan": 1, "view_plan": 2,
+                    "close_plan": 3,
+                    "return_main_menu": "r"}
+
+    def plan_menu(self) -> None:
+        print("\033[100m\033[4m\033[1mManage Emergency Plan\033[0m\n"
+              "[ 1 ] Create a new emergency plan\n"
+              "[ 2 ] View an emergency plan\n"
+              "[ 3 ] Close an emergency plan\n\n"
+              "[ R ] Return to Main Menu\n")
+
+    def precmd(self, option: str) -> str:
+        """
+        Transfer option numbers to function name for Admin Menu
+        """
+        if option.isdigit() and int(option) in list(self.plan_options.values()):
+            return list(self.plan_options.keys())[list(self.plan_options.values()).index(int(option))]
+        elif option.upper() == 'X' or option.upper() == 'R':
+            return list(self.plan_options.keys())[list(self.plan_options.values()).index(option.lower())]
+        else:
+            return "invalid_input"
+
+    def preloop(self) -> None:
+        """
+        Display plan menu.
+        :return:
+        """
+        self.plan_menu()
+
+    def return_previous_page(self) -> bool:
+        super().return_previous_page()
+
+    # ----- basic commands -----
+    def do_create_plan(self, arg):
+        """
+        #1 Create a plan
+        """
+        print("\033[100m\033[4m\033[1m{}\033[0m ".format("Create a new emergency plan"))
+        e_name = input("Enter emergency plan name:")
+        # -- handle emergency type
+        print("Enter a emergency type from the list")
+        emergency_types = plan_controller.list_emergency_types()
+        e_types_dict = {t.value: t for t in emergency_types}  # create a dictionary for Enum
+        for key in e_types_dict.keys():
+            print(f'- {key}')
+        while True:
+            e_type = input(">")
+            try:
+                emergency = e_types_dict[e_type]
+                e_type: Plan.EmergencyType = emergency
+                break
+            except KeyError:
+                print(f'Type {e_type} is not on the list. Please re-enter type:')
+
+        e_description = input("Enter description:")
+        e_geo_area = input("Enter graphical area:")
```

Comment:
Could this be changed to "Enter geographical area:" please?

Reply:
Thanks for spotting that! Typo has been corrected.